specification/consumption/resource-manager/Microsoft.Consumption/2017-12-30-preview/consumption.json

```diff
@@ -0,0 +1,456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-12-30-preview",
+    "title": "ConsumptionManagementClient",
```

Comment:
`title` should just be `"Consumption Management"`

Reply:
I think last time when it was changed @lmazuel Suggested it this way due to the requirement on the SDK to generate the client name correctly.

Reply:
@fearthecowboy Again, this is a C# behavior. Please move this "auto Client suffixing" to the modeler if you wish this behavior for everybody. In the meantime, changing that will break Python (at least, but probably not only)

Reply:
@lmazuel Right!  @sandeepnl

Reply:
Sure! We will leave it as is. Thanks!